### ISSUE TITLE
Adjust zoom behavior when clicking on a stop area depending on current zoom level

### ISSUE THREAD

**Issue by 7h30n3** [OPENER-next/OpenStop#146]
### Idea

Currently when a user is zoomed in on a stop area that extends over the bounding box/ viewport and clicks on that particular stop area the camera will zoom out so that the whole stop area will fit in the viewport.
This behavior can be distracting.

I suggest to only zoom in on a clicked stop area, when the camera is far away and never zoom out when the camera is already close and the clicked stop area fills the whole viewport.

tl;dr:
Don't zoom out to fit the stop area in the viewport, only zoom in.

**Comment by Robbendebiene** [comment]
Obsolete since #172
